Code/park_utils.py

```diff
@@ -92,3 +94,33 @@ def get_park_average_wait_times(park: Park) -> dict[str, float]:
 
         average_wait_times[attraction_name] = sum(queue_wait_list) / len(queue_wait_list)
     return average_wait_times
+
+def get_park_mean_and_std_attraction_visits(park: Park) -> Tuple[float, float]:
```

Comment:
I would rename this as park mean does not indicate what mean.
The functions gives standard deviation and mean for attraction visits.
Not park mean